pi_piper-sysfs.gemspec

```diff
@@ -0,0 +1,32 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'pi_piper/sysfs/version'
```

Comment:
This will work, the gemspec only needs to load `version.rb` which should only have the `VERSION` constant and nothing else:

`lib/sysfs/version.rb`:
```ruby
module PiPiper
  module Sysfs
    VERSION = '0.1.0'
  end
end
```

`lib/sysfs.rb`
```ruby
require 'pi_piper' # <- this is required first so PiPiper::Driver is now available in any other files below
require 'pi_piper/sysfs/version'
require 'pi_piper/sysfs/sysfs/ # <- This file can now use Sysfs < PiPiper::Driver
# ... any more requires

module PiPiper
  self.driver= PiPiper::Sysfs
  module Sysfs
    # ...
  end
end
```
`lib/sysfs/sysfs.rb`
```ruby
module PiPiper
  class Sysfs < PiPiper::Driver
     # ...
  end
end
```